src/aiq/front_ends/fastapi/fastapi_front_end_config.py

```diff
@@ -110,6 +110,9 @@ class CrossOriginResourceSharing(BaseModel):
     root_path: str = Field(default="", description="The root path for the API")
     host: str = Field(default="localhost", description="Host to bind the server to")
     port: int = Field(default=8000, description="Port to bind the server to", ge=0, le=65535)
+    ssl_certfile: str | None = Field(default=None, description="Path to the SSL certificate file")
+    ssl_keyfile: str | None = Field(default=None, description="Path to the SSL key file")
+    ssl_keyfile_password: str | None = Field(default=None, description="Password for the SSL key file")
```

Comment:
This option should be documented in https://github.com/NVIDIA/AIQToolkit/blob/develop/docs/source/reference/cli.md#fastapi